invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/user_dashboard/components/requests.js

```diff
@@ -266,10 +264,8 @@ export const RequestToggleComponent = ({
       return;
     }
     setOpen(false);
-    const filterToRemove = userSelectionFilters.filter((obj1) => {
-      return obj1.includes("is_open");
-    });
-    const filters = filterToRemove ? filterToRemove : [];
+    // We add the selected filters by the user to remove them
+    const filters = [...userSelectionFilters];
```

Comment:
Shouldn't that be a `let` as you mutate the variable just below?

Reply:
Hmm I think it can be a `const` since it does not stop array and objects from being modified it only stops the variable itself from being reassigned or being overwritten if I am not mistaken

Reply:
`const` is ok, if you add/remove items to an array will not change the obj. Assigning the const to a new array will fail indeed.